### ISSUE TITLE
Deal with bookdown better

### ISSUE THREAD

**Issue by elinw** [ropensci/skimr#473]
We have a few issues related to rendering in bookdown.  It's just a special case of rendering, but it's an important one, so I'd like to centralize here. 
#453 #424 ... what else?

**Comment by elinw** [comment]
What about if we made a compact print that didn't include the lines? 

Are we assuming the if someone is in bookdown they will organize the width themselves by selecting which statistics and potentially skimming twice to break it into outputs that fit on a page?

**Comment by elinw** [comment]
https://github.com/ropensci/skimr/pull/561

**Comment by srwerth** [comment]
I'm having trouble with skimr and bookdown as well. When I knit a document with the skim_without_charts() function, the table is too wide for the page and has no horizontal scrollbar.
![Screen Shot 2022-01-02 at 4 18 53 PM](https://user-images.githubusercontent.com/59977904/147890879-ac4dfc97-5b23-42d1-8e07-bbc0554ceda4.png)

**Comment by elinw** [comment]
Yes, and this relates to the issue of data with many columns. #370 I think we need to look closely at the print methods and see if we can add an option to allow nice splitting up into subtables beyond a set number of columns.  I think the Pander package does this.  I'm looking at thos means and sds also and wondering why they are rendering with so many columns, but that is probably a function of your data.  @michaelquinn32 what do you think?

**Comment by michaelquinn32** [comment]
kableExtra can support horizontal scrolling.
https://stackoverflow.com/questions/45849433/horizontal-scrolling-in-rmarkdown/45886734

This should be pretty easy to support after #667. We will modify the markdown rendering function, which is what is happening here, to include a horizontal scroll. Or the user can do it.
